resume-buddy/src/test/java/com/google/sps/CommentsTest.java

```diff
@@ -34,46 +35,38 @@
   @Before
   public void setUp() throws ParseException {
     helper.setUp();
+    uuid = UUID.randomUUID().toString();
     datastore = DatastoreServiceFactory.getDatastoreService();
     match = new Entity("Match");
     match.setProperty("reviewee", "shreyabarua@google.com");
     match.setProperty("reviewer", "sundar@google.com");
+    match.setProperty("uuid", uuid);
 
     comment1 = new Entity("Review-comments");
     comment1.setProperty("reviewer", "sundar@google.com");
     comment1.setProperty("reviewee", "shreyabarua@google.com");
     comment1.setProperty("text", "this is the first comment");
+    comment1.setProperty("matchID", uuid);
 
     comment2 = new Entity("Review-comments");
     comment2.setProperty("reviewer", "sundar@google.com");
     comment2.setProperty("reviewee", "shreyabarua@google.com");
     comment2.setProperty("text", "this is the second comment");
+    comment2.setProperty("matchID", uuid);
 
     comment3 = new Entity("Review-comments");
     comment3.setProperty("reviewer", "sundar@google.com");
     comment3.setProperty("reviewee", "shreyabarua@google.com");
     comment3.setProperty("text", "this is the third comment");
+    comment3.setProperty("matchID", uuid);
   }
 
   @After
   public void tearDown() {
     helper.tearDown();
   }
 
-  @Test
-  public void getMatch1() throws ParseException {
-    datastore.put(match);
-    String matchUser = CommentServlet.getMatch(UserType.REVIEWEE, "shreyabarua@google.com");
-    Assert.assertEquals("sundar@google.com", matchUser);
-  }
-
-  @Test
-  public void getMatch2() throws ParseException {
-    datastore.put(match);
-    String matchUser = CommentServlet.getMatch(UserType.REVIEWER, "sundar@google.com");
-    Assert.assertEquals("shreyabarua@google.com", matchUser);
-  }
-
+  // TODO: fix this test!
```

Comment:
Please add a GitHub Issue for this TODO and link it in the comment.

Reply:
Sorry this test was already fixed, I just forgot to remove the TODO. Got rid of it now!